src/compiler/crystal/tools/doc/type.cr

```diff
@@ -3,6 +3,13 @@ require "./item"
 class Crystal::Doc::Type
   include Item
 
+  PSEUDO_CLASS_PREFIX = "CRYSTAL_PSEUDO__"
+  PSEUDO_CLASS_NOTE   = <<-DOC
+
+    NOTE: This is a pseudo-class provided directly by the Crystal compiler.
+    It cannot be reopened nor overridden.
```

Comment:
```suggestion
    NOTE: This is a pseudo-class provided directly by the Crystal compiler.
          It cannot be reopened nor overridden.
```

Reply:
Don't think this should be added. The current format matches the pseudo-method note [here](https://github.com/crystal-lang/crystal/blob/0cef61e514267c265eb86f85ab250f85819e23c7/src/compiler/crystal/tools/doc/method.cr#L11), and the docs generator will remove the spacing regardless.

Reply:
I wasn't sure if the current format wouldn't split the `NOTE` message.

Reply:
@Sija I think it would be helpful to add a comment to auch a blank code suggestion. *Especially* id you're not sure about it.